apps/chess-training-app/src/components/training/tactics/create/TacticsSetCreator.tsx

```
@@ -0,0 +1,505 @@
+                onClick={() => setCreateFromPgn(true)}
+              >
+                From PGN
+              </Button>
+            </div>
+            {hasUnlimitedSets || setCount < maxSets ? (
+              <>
+                <div className="mb-4 flex flex-col gap-2">
+                  {!createFromPgn ? (
+                    <>
+                      <div className="">
+                        <label>Set Name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)